src/posts/announce-2.4.0.Final.adoc

```diff
@@ -0,0 +1,209 @@
+              flag: required
+              login-module-stack-ref: test-login-module-stack
+----
+
+It is now enough to write this:
+
+[source,yaml]
+----
+thorntail:
+  microprofile:
+    jwtauth:
+      realm: test-realm
+      roles:
+        file: role-mapping.properties
+----
+
+Internally, the second configuration is transformed to the first one, so the behavior is identical.
+
+== Gradle improvements
+
+Thorntail Gradle plugin has seen yet another round of improvements in 2.4.0.Final.
+
+== Changelog
+Release notes for 2.4.0.Final are available https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12317020&version=12339973[here].
+
+=== Feature Request
+* [https://issues.jboss.org/browse/THORN-1594[THORN-1594]] Verify/Fix execution on JDK 11 (Done)
+* [https://issues.jboss.org/browse/THORN-2324[THORN-2324]] provide the ability to toggle filtering of WEB-INF/lib (Done)
+
+=== Enhancement
+* [https://issues.jboss.org/browse/THORN-1852[THORN-1852]] MP JWT Fraction should integrate with the Keycloak client token introspector/verifier (Done)
+* [https://issues.jboss.org/browse/THORN-2313[THORN-2313]] Gradle plugin: provide tasks to run|start|stop the application (Done)
+* [https://issues.jboss.org/browse/THORN-2325[THORN-2325]] update default favicon with thorntail logo (Done)
+
+=== Task
+* [https://issues.jboss.org/browse/THORN-1933[THORN-1933]] MP FT - align CircuitBreaker.failOn() functionality once the spec issue is resolved (Done)
+* [https://issues.jboss.org/browse/THORN-2297[THORN-2297]] upgrade Keycloak to the latest 4.8.x.Final (Done)
+* [https://issues.jboss.org/browse/THORN-2339[THORN-2339]] Infinispan fraction should be added to supported fractions (Done)
+* [https://issues.jboss.org/browse/THORN-2352[THORN-2352]] Simplify microprofile-jwt configuration of the authentication module (Done)
+
+=== Epic
+* [https://issues.jboss.org/browse/THORN-2248[THORN-2248]] SmallRye MP 2.2 Impls (Done)
+
+=== Component Upgrade
+* [https://issues.jboss.org/browse/THORN-2251[THORN-2251]] Fault Tolerance 2.0 (Done)
+* [https://issues.jboss.org/browse/THORN-2256[THORN-2256]] REST Client 1.2 (Done)
+* [https://issues.jboss.org/browse/THORN-2259[THORN-2259]] OpenTracing 1.3 (Done)
+* [https://issues.jboss.org/browse/THORN-2260[THORN-2260]] OpenAPI 1.1 (Done)
+* [https://issues.jboss.org/browse/THORN-2340[THORN-2340]] update OpenShift REST Client Java to 7.0.0 (Done)
+
+=== Bug
+* [https://issues.jboss.org/browse/THORN-2287[THORN-2287]] Multi-module projects with files in the same directories, e.g. META-INF/beans.xml (Done)
+* [https://issues.jboss.org/browse/THORN-2307[THORN-2307]] build fails on JDK 11 (Done)
+* [https://issues.jboss.org/browse/THORN-2311[THORN-2311]] conflicting definitions of the org.eclipse.microprofile.restclient module (Done)
+* [https://issues.jboss.org/browse/THORN-2312[THORN-2312]] Runner fails with ZipException: duplicate entry: /WEB-INF/classes/META-INF/jandex.idx (Duplicate Issue)
```

Comment:
Maybe changing the message here to indicate that it fixes support for multi-module projects would be good?

Reply:
Autogenerated :-) But I know there's more fixes to Runner, do you want to write an extra section to the main announcement?

Reply:
I added a section about Runner fixes. I wanted to create a PR for it and instead I modified your branch, sorry

Reply:
That is absolutely OK and actually what I expected :-)